src-built-in/components/windowTitleBar/src/stores/windowTitleBarStore.js

```diff
@@ -9,7 +9,38 @@ var WindowClient;
 import windowTitleBarStoreDefaults from "./windowTitleBarStoreDefaults";
 import * as async from "async";
 var finWindow = fin.desktop.Window.getCurrent();
-//theses are constants that are set inside of setupStore. so they're declared as vars and not constantsa.
+
+//autohide functions and timers
+let headerTimeout = null;
+let suspendAutoHide = false;
+let autohideTimeout = 2000;
+let autohideSavedBodyMargin = null;
+const autoHideDefaultConfig = {
+	defaultSetting: false,
+	timeout: 2000,
+	resetMargin: true
+};
+let autoHideConfig = autoHideDefaultConfig;
+let autoHideTimer = function () {
```

Comment:
Function definitions should be const unless you plan to redefined them.

Reply:
done